src/Orchard.ResourceManagement/TagHelpers/StyleTagHelper.cs

```diff
@@ -114,6 +114,29 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 {
                     definition.SetDependencies(Dependencies.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries));
                 }
+
+                // Also include the style
+
+                var setting = _resourceManager.RegisterResource("stylesheet", Name);
+
+                if (At != ResourceLocation.Unspecified)
+                {
+                    setting.AtLocation(At);
+                }
+
+                setting.UseCdn(UseCdn);
```

Comment:
There is no issue here, it's just copying a parameter, it has to be handled when rendering the resource and throw.